cypress/integration/pages/articles/tests.js

```diff
@@ -49,41 +49,35 @@ export const testsThatFollowSmokeTestConfig = ({
 
     describe(`Article Body`, () => {
       it('should render a H1, which contains/displays a styled headline', () => {
-        cy.request(`${config[service].pageTypes.articles.path}.json`).then(
-          ({ body }) => {
-            const headlineData = getBlockData('headline', body);
-            cy.get('h1').should(
-              'contain',
-              headlineData.model.blocks[0].model.blocks[0].model.text,
-            );
-          },
-        );
+        cy.request(`${Cypress.env('currentPath')}.json`).then(({ body }) => {
```

Comment:
I remember this pattern was problematic in the past, as it prevented us from using querystrings

Is that still a problem?  If we were going to fix it, this would probably be the refactor to fix it, because it's all around dealing with URLs anyway

Fine to leave it as it's not entirely in scope though

Reply:
I'm keen to get this committed as is, and tackle the queryStrings issue if and when we need it. Ideally I would like to have the data look up in a single place - probably at the same time as we visit the page (which may or may not improve performance), but I think it's something we can fix later.